phpcs.xml.dist

```diff
@@ -15,23 +15,12 @@
         <!-- inapplicable for this project -->
         <exclude name="Generic.Strings.UnnecessaryStringConcat.Found" />
         <exclude name="PSR1.Classes.ClassDeclaration.MultipleClasses" />
-        <exclude name="PSR2.Classes.PropertyDeclaration.Underscore" />
```

Comment:
Are you sure about that? I added them manally after seeing errors in PHPStorm. Maybe it's because this ruleset only check Plugin.php (line 11) whereas PHPStorm was checking everything?

Reply:
Good point. Even though it's bad on PHPStorm's part to only partially respect configuration it still makes sense to add stubs and tests to phpcs checks.

Reply:
Are you adding them or do you want me to?